sanic/exceptions.py

```diff
@@ -169,6 +169,11 @@ def __init__(self, message, status_code=None, scheme=None, **kwargs):
             }
 
 
+class LoadModuleFromFileLocationException(ImportError):
```

Comment:
Inheriting from ImportError is OK, or should I pick some other exception ?

Reply:
This is OK

Reply:
I would subclass `SanicException`

Reply:
OK.